src/NHibernate/Type/OneToOneType.cs

```diff
@@ -44,7 +44,8 @@ public override void NullSafeSet(DbCommand st, object value, int index, bool[] s
 
 		public override void NullSafeSet(DbCommand cmd, object value, int index, ISessionImplementor session)
 		{
-			//nothing to do
```

Comment:
Is this change required? Hibernate did not change that.

Reply:
Same as this [comment](https://github.com/nhibernate/nhibernate-core/pull/2084/files#r269046031)